### ISSUE TITLE
PRs are sent to old GH account

### ISSUE THREAD

**Issue by novakov-alexey** [VirtusLab/scala-steward-repos#174]
Actual: Scala steward submits PRs to my old account: https://github.com/novakov-alexey-zz
Expected: Scala steward should submit PRs to my new account https://github.com/novakov-alexey

After my old account was locked, I requested old name for my new account. GitHub team changed my old account name, because account names must be unique at GitHub.

**Comment by tgodzik** [comment]
Thanks for reporting! This looks like an issue with the Scala steward itself and should be reported in https://github.com/scala-steward-org/scala-steward

**Comment by tgodzik** [comment]
So it should be fine if you delete the fork @fthomas  ?

**Comment by fthomas** [comment]
Yes, deleting @scala-steward's forks of @novakov-alexey's repos should fix things. I would prefer if someone else deletes these forks, because this kind of work is the reason why I've stopped maintaining the public Scala Steward instance myself.

@novakov-alexey it is probably helpful if you list [`gh repo delete`](https://cli.github.com/manual/gh_repo_delete) commands here to delete @scala-steward's forks of your repos so that anyone with the appropriate permissions can copy and paste that into their shell. Something like:
```
gh repo delete scala-steward/$repo1
gh repo delete scala-steward/$repo2
...
```

**Comment by novakov-alexey** [comment]
@fthomas these are the repos:

gh repo delete scala-steward/http4s-spnego
gh repo delete scala-steward/krb-operator2
gh repo delete scala-steward/krb-operator
gh repo deletescala-steward/http4s-spnego

Once they removed I would need to add them to [makrdown](https://github.com/VirtusLab/scala-steward-repos/blob/main/repos-github.md) file again, right?

**Comment by fthomas** [comment]
> Once they removed I would need to add them to [makrdown](https://github.com/VirtusLab/scala-steward-repos/blob/main/repos-github.md) file again, right?

There is no need to change the `repos.md` file. Once the forks are deleted, @scala-steward will fork your repos again on its next run.

**Comment by tgodzik** [comment]
I removed the forks @novakov-alexey let me know if the Scala Steward runs on the correct repos

**Comment by novakov-alexey** [comment]
@tgodzik thank you. I guess I need just to wait until any next PR from scala-steward? So far no incoming PRs on my side.

**Comment by tgodzik** [comment]
> @tgodzik thank you. I guess I need just to wait until any next PR from scala-steward? So far no incoming PRs on my side.

It's should run today or tomorrow, so let me know if you don't get any PR by monday.

**Comment by tgodzik** [comment]
Looks like that worked!

**Comment by novakov-alexey** [comment]
Thanks a lot guys! I have received PRs for one of the repository.
